LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 Peter Njuguna
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
```

Comment:
Optional  - it is advisable to commit often in your project to reduce ur chances of breaking your code and aids to backtrack bugs found in the code 👍🏾 💯

Reply:
Thank you.